<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPairTypeRegister</class>
 <widget class="QDialog" name="DlgPairTypeRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pair Type With Registers</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="styleSheet">
      <string notr="true">QHeaderView::section {    
	background-color: rgb(227, 227, 227);
    padding-left: 4px;
    border: 1px solid lightgrey;	
	font: 10pt &quot;MS Shell Dlg 2&quot;;
}

/*QTableView {
	background-color: white;
	show-decoration-selected: 0;
	padding: 0;
	margin: 0;
}*/

/*QTableView::item:selected
{   
    color: white;
    background: #0063cd;            
}*/

QTableView::item:selected:active
{
	color: white;
	background: #0063cd; 
   /* background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #6ea1f1, stop: 1 #567dbc);*/
}

QTableView::item:selected:!active 
{
	color: white;
	background: #0063cd;
   /* background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #6b9be8, stop: 1 #577fbf);*/
   /*background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ABAFE5, stop: 1 #8588B2);*/
}

QCheckBox {
	margin: 0px 5px 0 5px;
}

QCheckBox::focus {
	color: white;
}

*{outline:  none;}

</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableView" name="tblType">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QTableView" name="tblRegister">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="styleSheet">
      <string notr="true">* { button-layout: 2 }</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Discard</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
